NuRadioMC/test/SignalProp/T01test_python_vs_cpp.py

```diff
@@ -62,6 +70,26 @@
 t_python = time.time() - t_start
 print("Python time = {:.1f} seconds = {:.2f}ms/event".format(t_python, 1000. * t_python / n_events))
 
+
+
+if numba_available:
```

Comment:
I think this is potentially dangerous. We want the test to always test the numba implementation. In particular, our continuous CI on github should test it. I would prefer to remove the if clause and already fail when numba is not available.